ws-nextjs-app/pages/[service]/av-embeds/AvEmbedsMetadata.tsx

```diff
@@ -0,0 +1,88 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import { Helmet } from 'react-helmet';
+import filterForBlockType from '../../../../src/app/lib/utilities/blockHandlers';
+import buildAvEmbedURL from '../../../utilities/buildAvEmbedUrl';
+import { AvEmbedsPageProps } from './types';
+
+const AvEmbedsMetadata = ({ pageData }: AvEmbedsPageProps) => {
+  const { mediaBlock, metadata, promo } = pageData;
+  const promoSummary =
+    promo.summary.blocks[0].model.blocks[0].model.blocks[0].model.text;
```

Comment:
I'd use optional chaining here as we are accessing a deeply nested object.

With that added, it may be worth conditionally rendering the meta `description` in the event of it being `null / undefined`?

Reply:
Similarly with `promo.headlines.seoHeadline` as well, would be worth adding optional chaining as it can be `null`